dev/benchmarks/platform_views_layout/android/app/src/main/AndroidManifest.xml

```diff
@@ -20,5 +20,9 @@ found in the LICENSE file. -->
             </intent-filter>
         </activity>
         <meta-data android:name="flutterEmbedding" android:value="2"/>
+        <!-- Sample AdMob app ID: ca-app-pub-3940256099942544~3347511713 -->
+        <meta-data
+            android:name="com.google.android.gms.ads.APPLICATION_ID"
+            android:value="ca-app-pub-3940256099942544~3347511713" />
```

Comment:
This is the only change on top of https://github.com/flutter/flutter/pull/145224